src/hostedPages/activities/form/ConversationalForm.tsx

```diff
@@ -8,6 +8,7 @@ import layoutClasses from '../../layouts/HostedPageLayout/hostedPageLayout.modul
 import { FormProps } from '../../../types/form'
 import { useConversationalForm } from '../../../hooks'
 import { Question, UserQuestionType } from '../../../types'
+import { LoadActivityPlaceholder } from '../../layouts/HostedPageLayout/LoadActivityPlaceholder'
```

Comment:
np: we have 3 imports from same `../../layouts/HostedPageLayout` folder and an index file in it, can we please orgranize import so we import from that folder when we can instead from separate files.